### ISSUE TITLE
Panel updates too often for multi-file reporters

### ISSUE THREAD

**Issue by kaste** [SublimeLinter/SublimeLinter#1679]
In the performance PR #1673 I reverted https://github.com/SublimeLinter/SublimeLinter/commit/2cd32b7a6bc0d34b538aa8a71cd2930569da4c88 (and its successor https://github.com/SublimeLinter/SublimeLinter/commit/e803a44a38d70d7ac8581ff98f94858cd2df30d4) bc they looked complicated and unnecessary.  But we forgot that these commits weren't sole performance optimizations but visual optimizations.  Now, without them the panel (can) again look like this:

![aBj98NYsQl](https://user-images.githubusercontent.com/8558/68281535-2aa8c300-0078-11ea-9cef-a3635328628f.gif)

Basically, it looks like the errors streaming in and we just have too many updates which is totally ugly and not *just too slow*. 😞  

I mark as bug because it is really a visual bug.
